src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3916,4 +3913,13 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="ShowPluginsOverview_NonGlobal.Content" xml:space="preserve">
     <value>Not included in global results</value>
   </data>
+  <data name="PowerLauncher_TitleFontSize.Description" xml:space="preserve">
+    <value>The size of the result title and search query</value>
+  </data>
+  <data name="PowerLauncher_TitleFontSize.Header" xml:space="preserve">
+    <value>Text size</value>
```

Comment:
Might want to add the unit here. Note that in Windows Settings, they use percentage (100-225)

Reply:
Done!